tests/test_hf_hub_mixin.py

```diff
@@ -0,0 +1,177 @@
+import logging
+from pathlib import Path
+from typing import Any, Dict, Optional, Type, Union
+
+import pytest
+from huggingface_hub.hf_api import HfApi
+
+from pie_core.hf_hub_mixin import HFHubMixin
+from tests import FIXTURES_ROOT
+
+logger = logging.getLogger(__name__)
+
+CONFIG_PATH = FIXTURES_ROOT / "configs"
+WRONG_CONFIG_PATH = FIXTURES_ROOT
```

Comment:
don't use the default `FIXTURES_ROOT`. per coincidence, this may contain some file that can be loaded. I would propose to use a new temporary directory (e.g. via tmp_path).